honeybee_energy/ventcool/afn.py

```diff
@@ -0,0 +1,391 @@
+        the footprint.
+    """
+
+    xx, yy, zz = bounding_box_extents
+
+    hdist = 3 * max(xx, yy)
+    zdist = zz
+
+    return 'LowRise' if zdist <= hdist else 'HighRise'
+
+
+def compute_aspect_ratio(bounding_box_extents):
+    """Compute the AirflowNetwork aspect ratio of a building.
+
+    Args:
+        bounding_box_extents: # TBD
+    Returns:
+        A number representing the ratio of length of the short axis divided by the
+        length of the long axis.
+    """
+
+    xx, yy, _ = bounding_box_extents
+
+    if xx < yy:
+        return xx / yy
+    else:
+        return yy / xx
+
+
+def air_density_from_pressure(atmospheric_pressure=101325, air_temperature=20.0):
+    """Calculate density of dry air using the ideal gas law from temperature and pressure.
+
+    Args:
+        atmospheric_pressure: Atmospheric pressure in Pascals. (Default: 101325).
+        air_temperature: Air temperature in Celsius. (Default: 20.0).
+    Returns:
+        Air density at the given atmospheric pressure in kg/m3.
+    """
+    r_specific = 287.058  # specific gas constant for dry air in J/kg/K
+    air_temperature += 273.15  # absolute temperature
+    return atmospheric_pressure / (r_specific * air_temperature)
+
+
+def interior_afn(interior_face_groups, int_cracks):
+    """Mutate interior faces and subfaces to model airflow through cracks and openings.
+
+    This function creates an AFNCrack object with an air mass flow coefficient that
+    reflects the leakage characteristics of faces with different areas and types.
+    This requires multiplying the area-normalized air mass flow coefficients from the
+    reference crack data in int_cracks, by the wall area.
+
+    This function does not check adjacency information when computing leakage parameters
+    for adjacent faces or subfaces. It assumes adjacent faces share the same type and
+    area properties so that the computed leakage parameters are equivalent. Note that
+    EnergyPlus only requires one of the adjacent interzone surfaces to be
+    assigned a leakage component. If both adjacent surfaces have a leakage component,
+    the air flow through the surface will be counted twice.
+
+    Args:
+        interior_face_groups: A tuple with four groups of interior faces types:
+
+            * int_walls: List of interior Wall type Face objects.
+            * int_floorceilings: List of interior RoofCeiling and Floor type Face
+                objects.
+            * int_apertures: List of interior Aperture subface Face objects.
+            * int_doors: List of interior Door subface Face objects.
+
+        int_cracks: A dictionary of air mass flow coefficient and exponent data
+            corresponding to the face types in the interior_face_groups. Face
+            data flow coefficients should be normalized by surface area, and closed
+            opening flow coefficients should be normalized by edge lengths, for example:
+
+            {
+                # kg/s/m2 @ 1 Pa
```

Comment:
Not a correct dosctring.  See all of the other places where we include sample dictionaries in docstrings.